sdk/src/modules/account.ts

```diff
@@ -190,39 +194,56 @@ export default class Account {
    *        signing step.
    * @param messageSignedStepCallback A callback triggered after the message
    *        signing step.
-   * @returns Hash of the withdrawal transaction and the redemption key.
+   * @returns Hash of the withdrawal transaction and the redemption request id.
    */
   async initializeWithdrawal(
     btcAmount: bigint,
     dataBuiltStepCallback?: DataBuiltStepCallback,
     onSignMessageStepCallback?: OnSignMessageStepCallback,
     messageSignedStepCallback?: MessageSignedStepCallback,
-  ): Promise<{ transactionHash: string; redemptionKey: string }> {
+  ): Promise<{ transactionHash: string; redemptionRequestId: bigint }> {
     const tbtcAmount = fromSatoshi(btcAmount)
     const shares = await this.#contracts.acreBTC.convertToShares(tbtcAmount)
     // Including fees.
-    const redeemedTbtc = await this.#contracts.acreBTC.previewRedeem(shares)
-
-    const redeemerProxy = new OrangeKitTbtcRedeemerProxy(
-      this.#contracts,
-      this.#orangeKitSdk,
-      {
-        bitcoinAddress: this.#bitcoinAddress,
-        ethereumAddress: this.#ethereumAddress,
-        publicKey: this.#bitcoinPublicKey,
-      },
-      this.#bitcoinProvider,
+    // TODO: Do we need this?
+    // const redeemedTbtc = await this.#contracts.stBTC.previewRedeem(shares)
```

Comment:
No, it's required in the finalization when requesting unminting from the tBTC Bridge.

Reply:
fa7aa3e